aim/cli/convert/processors/tensorboard.py

```diff
@@ -191,24 +198,30 @@ def get_level(current_path):
                             unsupported_plugin_noticed = True
                         continue
 
-                    if plugin_name == 'images':
-                        tensor = value.tensor.string_val[2:]
-                        track_val = [
-                            Image(tf.image.decode_image(t).numpy()) for t in tensor
-                        ]
-                        if len(track_val) == 1:
-                            track_val = track_val[0]
-                    else:
-                        d_type = tf.dtypes.DType(value.tensor.dtype)
-                        decoded = tf.io.decode_raw(value.tensor.tensor_content, d_type)
-                        track_val = float(decoded)
+                    try:
+                        if plugin_name == 'images':
+                            tensor = value.tensor.string_val[2:]
+                            track_val = [
+                                Image(tf.image.decode_image(t).numpy()) for t in tensor
+                            ]
+                            if len(track_val) == 1:
+                                track_val = track_val[0]
+                        else:
+                            proto = tf.make_tensor_proto(value.tensor)
+                            track_val = tf.make_ndarray(proto)
+                    except Exception:
+                        # catch all the nasty failures
+                        continue
 
                     run_tb_log['values'][value_id] = {
                         'step': step,
                         'timestamp': timestamp
                     }
                     run._track_impl(track_val, timestamp, tag, step, context=event_context)
 
+                if fail_count:
```

Comment:
shoudn't fail_count be incremented in case of exception? also user must see the reason of failure (in case of multiple failures, show the first)